### DISCUSSION TITLE
How do I increase the size of tmpfs for private-opt?

### DISCUSSION THREAD

**Discussion by ghost** [netblue30/firejail#5307] [Q&A] [answered]
Hello,
When I try to start microsoft-edge-stable in firejail, I'm facing the issue where private-opt is being a too small tmpfs.
My /opt/microsoft directory is larger than 500MB.
I know it is not possible to give a subdir for private-opt.
Is it possible to increase the tmpfs size for private-opt instead of just ignoring it?

**Comment by ghost** [comment, marked as answer]
You can try increasing `file-copy-limit` in /etc/firejail/firejail.config.

**Reply by ghost** [reply]
I did try and it works, that's nice!

**Comment by rusty-snake** [comment]
Do you really want to copy >0,5GB into your RAM?

**Reply by ghost** [reply]
That's true, I could set a whitelist instead? Like
whitelist /opt/microsoft
whitelist /opt/microsoft/msedge

**Reply by ghost** [reply]
Maybe could we update the microsoft-edge profile to not add it to a tmpfs?

**Reply by rusty-snake** [reply]
> That's true, I could set a whitelist instead? Like whitelist /opt/microsoft whitelist /opt/microsoft/msedge

Yes,

```
ignore private-opt
whitelist /opt/microsoft/msedge
```

in `microsoft-edge.local` should work.



> Maybe could we update the microsoft-edge profile to not add it to a tmpfs?

Yes, makes sense IMHO.

Also #4742 FTR.

**Reply by ghost** [reply]
I did write in microsoft-edge.local:
```
ignore private-opt
whitelist /opt/microsoft/msedge
```
and it seems to apply the same restrictions private-opt does without tmpfs.

Edit: you answered before I did a refresh of that page.
